pkg/proto/nbcontract/v1/kubeletconfig.proto

```diff
@@ -1,15 +1,18 @@
 syntax = "proto3";
 package nbcontract.v1;
+import "pkg/proto/nbcontract/v1/featurestate.proto";
 
-message KubeletConfiguration {
+message KubeletConfig {
   // TODO(ace): remove these/make api defensible
-  repeated string kubelet_flags = 1;
+  map<string, string> kubelet_flags = 1;
   map<string, string> kubelet_node_labels = 2;
   repeated Taint taints = 3;
   repeated Taint startup_taints = 4;
   KubeletDiskType kubelet_disk_type = 5;
-  bool kubelet_config_file_enabled = 6;
+  FeatureState kubelet_config_file_status = 6;
   string kubelet_config_file_content = 7;
```

Comment:
Need a consistent pattern for this ... I presume "optional string kube_config_file_content" is not good enough for some reason?

Reply:
Sorry, are you referring to kubelet_config_file_status or string kube_config_file_content, or both? Could you please elaborate on consistent pattern here?

Reply:
I mean for a pair together - FeatureState indicating if something is present, paired with associated data. Elsewhere you have used separate messages for this scenario, I think.

Reply:
Oh you are referring to the status, not the content. Actually I should put a comment here that we will remove this status because it's only checking if the file content is present. I've noted it the design doc but let me add a comment here too.